data/sea/54-zebra.h

```diff
@@ -154,68 +185,142 @@ static ierror_msg_t zebra_translate
     return 0;
 }
 
-static int64_t zebra_translate_table
+/* Translate Zebra enum tags to Icicle bool flags, e.g. is_some, is_right, etc.
+   We only allow at most two variants since Icicle's runtime representation
+   of enums is unsuitable for more.
+ */
+static uint64_t zebra_translate_column_tags
+    ( const int64_t variant_count
+    , void **dst
+    , const int64_t *src )
+{
+    assert (variant_count <= 2);
```

Comment:
Is the intention that this is `assert` instead of `IASSERT` ?

If the intention is `assert` maybe we'd be better off saying `if (variant_count <= 2)` then printing a descriptive error and exiting.